lib/elixir/lib/version.ex

```diff
@@ -64,23 +66,28 @@ defmodule Version do
   `~> 2.0`       | `>= 2.0.0 and < 3.0.0`
   `~> 2.1`       | `>= 2.1.0 and < 3.0.0`
 
-  When `allow_pre: false` is set, the requirement will not match a
-  pre-release version unless the operand is a pre-release version.
+  The requirement operand after the `~>` is allows to omit the PATCH part of the version,
+  allowing us to express `~> 2.1` or `~> 2.1-dev`, something that wouldn't be allowed
+  when using the common operators.
```

Comment:
"when using common operators."?

Reply:
it mentions them at the beginning of the sections. I am referring to them as now "common comparison operators".
Or should we refer to the as "regular" or "standard" comparison operators?